packages/app/src/__deprecated__/passcode-modal/components/passcode-modal.test.tsx

```diff
@@ -116,7 +116,7 @@ test("Message is displayed properly when request about phone lock return interna
   )
 })
 
-test("Message is displayed properly when request about phone lock status return phone locked", async () => {
+test.skip("Message is displayed properly when request about phone lock status return phone locked", async () => {
```

Comment:
how about this skip? :)

Reply:
Thanks, Patryk for these comments. Skips will be handled as part of this task https://appnroll.atlassian.net/browse/CP-2234